Duckling/Numeral/DE/NumParser.hs

```diff
@@ -0,0 +1,154 @@
+module Duckling.Numeral.DE.NumParser (parseNumeral) where
+
```

Comment:
```suggestion

import Prelude
```

This code is synced into an internal repository where implicit Prelude is turned off; as a result the `instance Functor` can't be committed without an explicit Prelude import.

Reply:
I see. Fixed.